graphql/integration_atomic_test.go

```diff
@@ -55,6 +56,8 @@ func setup(t *testing.T, directory string) atomicGraphQLState {
 	env := evergreen.GetEnvironment()
 	ctx := context.Background()
 	require.NoError(t, env.DB().Drop(ctx))
+	logsDb := env.Client().Database(model.TaskLogDB)
```

Comment:
I am unsure if using the log db and task_logg collection is the best practice here. Currently we don't load any smoke data into the log db but if we did in the future it would be overwritten when running the gql test suite. The logsDb in the test environment should probably map to something like `mci_test_logs` and should be reflected here when getting the db https://github.com/evergreen-ci/evergreen/blob/c905b1bea0fddfeab8fa10ad0f40a8db8fdc349f/model/task_log.go#L51. Does anyone know how I can initialize this mapping in Environment or Session?

Reply:
This has the additional side effect of dropping other data in the logs db that the test is not using, but the person running the test may need for the purposes of testing locally. Instead of dropping the db, let's remove the specific document IDs that you're using in the test